packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/elements.ts

```diff
@@ -1,6 +1,11 @@
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 import Color from 'color';
 
+const svg = props =>
```

Comment:
For later: oh noes, would be cool to drop this in a file so it's discoverable?

Reply:
Honestly I don't think we will ever used this again, I could have done in css now that I think of it